src/container-client/ContainerClientPlugin.js

```diff
@@ -84,6 +84,21 @@
 		 */
 		this.playOptions = {};
 
+		// attempt to load play options from the query string
+		var match = /playOptions=[^&$]*/.exec(window.location.search);
+		if (match !== null)
+		{
+			var matchedToken = match[0];
+			var rawValue = decodeURIComponent(matchedToken.split('=')[1]);
+
+			try
+			{
+				this.playOptions = JSON.parse(rawValue);
+			}
+			catch (e)
+			{}
```

Comment:
could we add a console.error in the catch? That way it's at least doing something

Reply:
Done.